ci/jenkins/checkin.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
May be a short description on what each of the script does at the beginning will help